queue_processors/queue_processor/handle_message.py

```diff
@@ -181,7 +181,7 @@ def reduction_started(self, message: Message):
                 rb_number=message.rb_number, run_number=message.run_number,
                 run_version=message.run_version)
 
-        if reduction_run.status.value not in ("Error", "Queued"):
+        if reduction_run.status.value not in ['e', 'q']:
```

Comment:
Would prefer not to use short hand in code base it's less obvious what is going on.

Reply:
I tried this with `.verbose_value()` but it didn't work because it's being mocked I think. I could add comments explaining the verbose values in these instances?